test/integration/tools/issue_test.py

```diff
@@ -0,0 +1,92 @@
+from unittest.mock import patch
+
+from exasol.toolbox.tools.issue import CLI
+
+
+class TestListIssues:
```

Comment:
Is there a specific reason for the class?

Reply:
After reading more of these classes in this PR, I actually agree with you. It was actually quite nice to review.